cms/djangoapps/contentstore/views/course.py

```diff
@@ -980,7 +980,7 @@ def settings_handler(request, course_key_string):
             about_page_editable = not marketing_site_enabled
             enrollment_end_editable = GlobalStaff().has_user(request.user) or not marketing_site_enabled
             short_description_editable = settings.FEATURES.get('EDITABLE_SHORT_DESCRIPTION', True)
-
+            enable_extended_course_details = settings.FEATURES.get('ENABLE_EXTENDED_COURSE_DETAILS', False)
```

Comment:
We will need to use the microsite module in order to check for this value. See line 974 above.